Essentials/src/main/java/com/earth2me/essentials/utils/StringUtil.java

```diff
@@ -8,6 +8,7 @@
 public final class StringUtil {
     private static final Pattern INVALIDFILECHARS = Pattern.compile("[^a-z0-9-]");
     private static final Pattern STRICTINVALIDCHARS = Pattern.compile("[^a-z0-9]");
+    @SuppressWarnings("CheckStyle")
```

Comment:
If you're trying to suppress something from Checkstyle, you should be suppressing individual checks, not everything. https://checkstyle.org/config_filters.html#SuppressWarningsFilter might be useful

Reply:
I don't want to fully suppress it, it's just this one place